src/components/SentimentAnalysisGraph.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { VictoryAxis } from 'victory';
+import { VictoryBar } from 'victory';
+import { VictoryChart } from 'victory';
+
+class SentimentAnalysisGraph extends Component {
+	static displayName = 'SentimentAnalysisGraph';
+
+	state = {
+		callFunc: this.getData(),
+		data: {pos_value: 90, neg_value: 10}
+	};
+
+	getData(){
```

Comment:
I like the animation :) but this component shouldn't be concerned with modifying or creating the data driving the graph. It should just render whatever its parent sends via props.